File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set(nums)
        print(s)
        output = 0

        for n in nums:
            curr = 0
            if n - 1 not in s:
                # this is the start of a sequence
                i = n
                while i in s:
                    curr += 1
                    i += 1
                if curr > output:
                    output = curr

        return output 

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean output = false;
        for(int i = 0; i < nums.length; i++){
            int numCurr = nums[i];
            for(int j = 0; j < nums.length; j++){
                if(nums[j] == numCurr & i != j){
                    output = true;
                }

            }
        }
        return output;
    }
}
